### PR TITLE
support for --adapter and --adapters flag added

### LINKED ISSUE #6997: Error: unknown flag: --adapter in mesheryctl command
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
Error: unknown flag: --adapter.   

```
$ mesheryctl system check --adapter meshery-istio:10000

Error: unknown flag: --adapter
```

#### Desired Behavior
--adapter is supposed to be a valid command based on the official documentation.  

Link to the official documentation:  https://docs.meshery.io/reference/mesheryctl/system/check.  

#### Screenshots/Logs
Screenshot 2023-02-03 at 11 57 32 AM

#### Environment

- Host OS: Mac 
- Meshery Server Version: v0.6.47	4c357f30
- Meshery Client Version: v0.6.49	460e9bd43
- Platform: Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by deepakdinesh1123**
Signed-off-by: deepakdinesh1123 <d.deepakdinesh13@gmail.com>

**Notes for Reviewers**

This PR fixes #6997 

I have added the support for `--adapter` and `--adapters` flag to check if adapters are working properly.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1e18abbac87cc848e817faa0231da4dff05a1a9c at: https://63e7563052e11b06b5338e7c--meshery-docs-preview.netlify.app

**Comment by Chadha93** [comment]
@deepakdinesh1123  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by leecalcote** [comment]
Re-emphasizing @Chadha93's suggestion, @deepakdinesh1123 it would be nice to see this one demo'ed.

@alphaX86 what considerations need to be made here in context of how `--adapter` flag reconciles with meshconfig?

File: mesheryctl/internal/cli/root/system/check.go [hunk 1]
```diff
@@ -477,65 +488,47 @@ func (hc *HealthChecker) runComponentsHealthChecks() error {
...
+// runOperatorHealthChecks executes health-checks for Operators
+func (hc *HealthChecker) runOperatorHealthChecks() error {
+	//TODO
+	return nil
 }
 
 // runAdapterHealthChecks executes health-checks for Adapters
-func (hc *HealthChecker) runAdapterHealthChecks() error {
+// If no adapter is specified all the adapters are checked
+func (hc *HealthChecker) runAdapterHealthChecks(adapter_name string) error {
```
Comment (hexxdump) [inline mesheryctl/internal/cli/root/system/check.go:502]: pll fix the lint-error

Comment (leecalcote) [reply mesheryctl/internal/cli/root/system/check.go:502]: @deepakdinesh1123 you might have outstanding feedback here.

Comment (deepakdinesh1123) [reply mesheryctl/internal/cli/root/system/check.go:502]: I was thinking of demoing this first to get everyone's thoughts and then get back to fixing it. I also have another item to discuss along with this.

**Comment by Chadha93** [comment]
@deepakdinesh1123  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Pull request merged**